### DISCUSSION TITLE
After starting coffee editor with docker on CentOS, open the interface and report an error

### DISCUSSION THREAD

**Discussion by 136861535** [eclipse-emfcloud/emfcloud#149] [Coffee Editor]
Open 3000 port in browser：
![图片](https://user-images.githubusercontent.com/101307516/158930092-29c3ee38-c045-4ac4-afdf-e1f56f82d02b.png)

Back end log information：
![图片](https://user-images.githubusercontent.com/101307516/158930408-55f28089-4a8d-46cd-94a8-4b6b17a9b43d.png)

However, this directory exists in this path of docker container：
![图片](https://user-images.githubusercontent.com/101307516/158930430-1a7a03c9-c27a-4491-ba47-02e9ffd66179.png)

Why？

**Comment by 136861535** [comment]
The first to enter the interface is good, but refreshing and entering again will report the above error.

**Reply by eneufeld** [reply]
Hi @136861535 ,
so the error in the first screenshot is weird as it points to a vsix file and none of the part of the coffee editor are a vscode extension. I never saw such an error.
I have no idea why it can't find the file on the second reload. @ndoschek any idea?

**Comment by 136861535** [comment]
Is there an error free version on GitHub




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Eugen ***@***.***&gt;; 
发送时间: 2022年3月18日(星期五) 下午3:53
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [eclipse-emfcloud/emfcloud] After starting coffee editor with docker on CentOS, open the interface and report an error (Discussion #149)





 
Hi @136861535 ,
 so the error in the first screenshot is weird as it points to a vsix file and none of the part of the coffee editor are a vscode extension. I never saw such an error.
 I have no idea why it can't find the file on the second reload. @ndoschek any idea?
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
